react/ProductListContext.tsx

```diff
@@ -39,7 +39,7 @@ export type Dispatch = (action: ReducerActions) => void
 
 const DEFAULT_STATE: State = {
   nextImpressions: [],
-  sentIds: new Set<string>(),
+  sentIds: new Map<string, { position: number }>(),
```

Comment:
Not using the position stored here, but thought it could be useful in the future